### PR TITLE
citra_qt: Move CPU JIT toggle to Debug tab.

### PR DESCRIPTION
**Pull request by adityaruplaha**
Self-explanatory.

---
Will attach an actual screenshot as soon as AppVeyor gives me the artifact. :P

image

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3971)

### CODE REVIEW

File: src/citra_qt/configuration/configure_debug.ui [hunk 1]
```diff
@@ -113,6 +113,13 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QCheckBox" name="toggle_cpu_jit">
+     <property name="text">
+      <string>Enable CPU JIT</string>
+     </property>
+    </widget>
+   </item>
```
Comment (zhaowenlan1779) [inline src/citra_qt/configuration/configure_debug.ui]: This won't look good imo, you've got to get a GroupBox

Comment (adityaruplaha) [reply src/citra_qt/configuration/configure_debug.ui]: For just 1 setting? And what should I name it?

Comment (zhaowenlan1779) [reply src/citra_qt/configuration/configure_debug.ui]: Name it whatever. CPU? Performace? Just do not leave it alone.

**Comment by adityaruplaha** [comment]
Because only 1 setting actually is related to CPU, and its a place where other setting like Shader JIT can also be moved as its a misc groupbox.

**Pull request merged**